ts/components/services/ServiceSectionListComponent.tsx

```diff
@@ -0,0 +1,93 @@
+import * as pot from "italia-ts-commons/lib/pot";
+import * as React from "react";
+import {
+  ListRenderItemInfo,
+  RefreshControl,
+  SectionList,
+  SectionListData,
+  StyleSheet
+} from "react-native";
+
+import { H3, ListItem } from "native-base";
+import { ServicePublic } from "../../../definitions/backend/ServicePublic";
+import { ProfileState } from "../../store/reducers/profile";
+import variables from "../../theme/variables";
+import { ServiceListItem } from "./ServiceListItem";
+
+type OwnProps = {
+  // Can't use ReadonlyArray because of the SectionList section prop
+  // typescript definition.
+  // tslint:disable-next-line:readonly-array
+  sections: Array<SectionListData<pot.Pot<ServicePublic, Error>>>;
+  profile: ProfileState;
+  refreshing: boolean;
```

Comment:
we usually prepend `is` to boolean variables

```suggestion
  isRefreshing: boolean;
```

Reply:
in the PR i fixed that